src/Symfony/Bridge/Twig/Extension/TranslationExtension.php

```diff
@@ -104,17 +104,17 @@ public function getTranslationNodeVisitor(): TranslationNodeVisitor
     }
 
     /**
-     * @param ?string|Translatable $message The message id (may also be an object that can be cast to string)
+     * @param string|\Stringable|Translatable|null $message
      */
     public function trans($message, array $arguments = [], string $domain = null, string $locale = null, int $count = null): string
     {
         if ($message instanceof Translatable) {
             $arguments += $message->getParameters();
-            $domain = $message->getDomain();
+            $domain = $domain ?? $message->getDomain();
```

Comment:
we reasoned the domain is bound to a translatable, and is not intended to be overridden dynamically.

So personally i'd prefer `$message->getDomain ?? $domain`, but this needs to be re-evaluted perhaps.

Reply:
fine to me, I've swapped these